src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -345,6 +346,10 @@ def get(self, request: Request, organization) -> Response:
         self.add_cursor_headers(request, response, cursor_result)
 
         # TODO(jess): add metrics that are similar to project endpoint here
+        transaction = Hub.current.scope.transaction
+        if transaction:
+            transaction.set_measurement("issues.unresolved", len(cursor_result))
```

Comment:
I could be wrong here since I'm not in issues code very much, but if a user visits the `For Review` tab directly via url (ie `/organizations/sentry/issues/?query=is%3Aunresolved+is%3Afor_review+assigned_or_suggested%3A%5Bme%2C+none%5D&sort=inbox`), wouldn't this measurement become inaccurate? It's reporting for `issues.unresolved` but the actual measurement should be for `issues.for_review`.